### DISCUSSION TITLE
How do you set the valueKey in data? Using CheckTree.

### DISCUSSION THREAD

**Discussion by PeterMountainPass** [rsuite/rsuite#1814] [Q&A]
Do you set the valueKey in json file?

If yes, how do you set the valueKey?

I am trying to make some of the data using uncheckableItemValues porps to disable the checkbox.

**Comment by SevenOutman** [comment]
`valueKey` is the property name in a object which you expect to present the "value" of that object.

For example, if you have a data object like

```json
{
  "id": 1,
  "name": "Admin"
}
```

and wants to output it's id 1 when selecting this option, you can set `valueKey="id"`.
